Patch.py

```diff
@@ -304,7 +304,7 @@ def read_rom(stream, strip_header=True) -> bytearray:
                     print(f"Creating patch for {rom}")
                     result = pool.submit(create_patch_file, rom, address)
                     result.add_done_callback(lambda task: print(f"Created patch {task.result()}"))
-
+                # TODO: @berserker does this if tree require updating to refer to new extensions?
```

Comment:
It *could* be added. Though running Patch.py directly has kind of fallen out of general use, we don't even freeze it for users.